src/main.cpp

```diff
@@ -1 +1,13 @@
-int main() {}
+#include <iostream>
+#include "export.h"
+
+constexpr uint32_t IMAGE_WIDTH = 512
+constexpr uint32_t IMAGE_HEIGHT = 512
+constexpr uint32_t INPUT_EXTENT = 1
+constexpr uint32_t OUTPUT_EXTENT = 32
```

Comment:
Screaming case is conventionally reserved for C macros and the like, use `image_width` or `image_width_v` (`v` for `value`, you'll see it in the standard library too, eg `std::pi_v<float>`).